### DISCUSSION TITLE
Make a new folder => do without confirmation

### DISCUSSION THREAD

**Discussion by treborxy** [zim-desktop-wiki/zim-desktop-wiki#2160] [Ideas]
Hello,

as I'm working with folders accompanying almost every page, the permanent question "Do you really want to make a new folder? Yes / No" is a bit annoying.
My sugestion: either make the new folder without confirmation, or offer a "never ask again" option (or similar).

Thank you once again for zim! :-)

Regards, Robert

**Comment by jaap-karssenberg** [comment]
This option is already in the new release (to be announced this week)

Op vr 30 sep. 2022 15:51 schreef treborxy ***@***.***>:

**Reply by treborxy** [reply]
Wow - perfect! Jaap rulez!! :-)
